src/main/java/org/apache/commons/crypto/utils/NativeCodeLoader.java

```diff
@@ -231,7 +231,7 @@ public static String getVersion() {
                 .getResource("/META-INF/maven/org.apache.commons.crypto/commons-crypto/pom.properties");
         if (versionFile == null) {
             versionFile = NativeCodeLoader.class
-                    .getResource("/org/apache/commons/crypto/VERSION");
```

Comment:
Do we need this change? It seems  {{.getResource("/org/apache/commons/crypto/VERSION");}} is right.